{
    "title" : "Rebuild index error",
    "uri" : "http://answers.coveo.com/questions/12100/rebuild-index-error.html",
    "printableUri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
    "clickUri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
    "uniqueId" : "42.4247$http://answers.coveo.com/questions/12100/rebuild-index-error.html",
    "excerpt" : "I got the error below: ... Any errors in the diagnostic page? ... Yes. ... It was the user name/password for the service admin account. ... It worked fine now. ... I guess something wrong with my S...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1206,
    "percentScore" : 78.143005,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "Rebuild index error",
      "sysauthor" : "qch1595",
      "sysurihash" : "rbp3B6yTñqyð7Hxg",
      "urihash" : "rbp3B6yTñqyð7Hxg",
      "numberofanswers" : 2,
      "sysuri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
      "sysprintableuri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
      "sitemappriority" : 0.1,
      "systransactionid" : 160367,
      "questionaskeddatestring" : "·\n            Jul 20, 2017 at 07:17 PM ·",
      "tags" : [ "rebuild index" ],
      "sysconcepts" : "root cause ; log file ; admin account ; Sitecore ; Anonymous ; Share",
      "concepts" : "root cause ; log file ; admin account ; Sitecore ; Anonymous ; Share",
      "printableuri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
      "documenttype" : "WebPage",
      "sysindexeddate" : 1517943247000,
      "permanentid" : "0f688cb109e01be19cb4b9a3508ebc643d10027416a88093add3e3d6c66a",
      "syslanguage" : [ "English" ],
      "transactionid" : 160367,
      "title" : "Rebuild index error",
      "date" : 1500854400000,
      "objecttype" : "Answers",
      "sitemapchangefrequency" : "Monthly",
      "answerauthor" : [ "qch1595", "qch1595" ],
      "hasanswer" : "true",
      "audience" : [ "Administrator", "Developer" ],
      "commentauthor" : [ "François Lachance-Guillemette \u2666\u2666", "François Lachance-Guillemette \u2666\u2666" ],
      "hasacceptedanswer" : "true",
      "sourcetype" : "Sitemap",
      "sysconnectortype" : "SitemapCrawler",
      "rowid" : 5449080,
      "numberofcomments" : 2,
      "size" : 81093,
      "sysdocumenttype" : "WebPage",
      "questionvotes" : 0,
      "clickableuri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
      "syssource" : "AnswersCloud",
      "orderingid" : 212384744911579360,
      "acceptedanswers" : 1,
      "syssize" : 81093,
      "sysdate" : 1500854400000,
      "author" : "qch1595",
      "source" : "AnswersCloud",
      "questionbodyhtml" : "<div class=\"question-body\">\n                <p>\n<p>I installed Coveo for Sitecore 4.0 without any issues. Then I login to Sitecore-> indexing manager to rebuild the index. I got the error below:</p>\n<pre>Job started: Index_Update_IndexName=Coveo_master_index|#Exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ArgumentNullException: Precondition failed: The parameter 'p_Communication' must not be null<br>Parameter name: p_Communication<br>  at Coveo.Framework.CNL.Precondition.NotNull(Object p_Parameter, String p_ParameterName)<br>  at Coveo.SearchProvider.AbstractProviderUpdateContext..ctor(ISearchIndexSummary p_IndexSummary, IIndexCommunication p_Communication)<br>  at Coveo.SearchProvider.ProviderUpdateContext..ctor(ISearchIndex p_Index, IIndexCommunication p_Communication)<br>  at Coveo.SearchProvider.ProviderIndex.CreateUpdateContext()<br>  at Coveo.SearchProvider.ProviderIndex.<Rebuild>b__3e()<br>  at Coveo.SearchProvider.ProviderIndex.TryPerformIndexingOperation(IndexingOptions p_IndexingOptions, Action p_Action)<br>  at Coveo.SearchProvider.ProviderIndex.Rebuild()<br>  --- End of inner exception stack trace ---<br>  at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)<br>  at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)<br>  at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)<br>  at Sitecore.Reflection.ReflectionUtil.InvokeMethod(MethodInfo method, Object[] parameters, Object obj)<br>  at Sitecore.Jobs.JobRunner.RunMethod(JobArgs args)<br>  at (Object , Object[] )<br>  at Sitecore.Pipelines.CorePipeline.Run(PipelineArgs args)<br>  at Sitecore.Pipelines.DefaultCorePipelineManager.Run(String pipelineName, PipelineArgs args, String pipelineDomain)<br>  at Sitecore.Jobs.Job.ThreadEntry(Object state)</pre> \n<p>\n                </div>",
      "sitemaplastmodified" : 1500854400000,
      "collection" : "default",
      "syssourcetype" : "Sitemap",
      "indexeddate" : 1517943247000,
      "questionnumber" : 12100,
      "connectortype" : "SitemapCrawler",
      "filetype" : "html",
      "sysclickableuri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
      "sysfiletype" : "html",
      "language" : [ "English" ],
      "sitemapparenturl" : "http://answers.coveo.com/questions/1.xml",
      "questionauthor" : "qch1595",
      "sitename" : "Answers",
      "sysrowid" : 5449080,
      "uri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
      "syscollection" : "default",
      "sitelanguage" : [ "English" ]
    },
    "Title" : "Rebuild index error",
    "Uri" : "http://answers.coveo.com/questions/12100/rebuild-index-error.html",
    "PrintableUri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
    "ClickUri" : "https://answers.coveo.com/questions/12100/rebuild-index-error.html",
    "UniqueId" : "42.4247$http://answers.coveo.com/questions/12100/rebuild-index-error.html",
    "Excerpt" : "I got the error below: ... Any errors in the diagnostic page? ... Yes. ... It was the user name/password for the service admin account. ... It worked fine now. ... I guess something wrong with my S...",
    "FirstSentences" : null
  }